pkg/kando/location_delete.go

```diff
@@ -46,6 +47,12 @@ func runLocationDelete(cmd *cobra.Command) error {
 	return locationDelete(ctx, p, s)
 }
 
+// kopiaLocationDelete pulls the data from a kopia snapshot into the given target
```

Comment:
Is this comment accurate? It does not match the function name though.

Reply:
No, it is wrong, let me fix that. Thanks for noticing.

Reply:
Fix: https://github.com/kanisterio/kanister/pull/967